services/painting-surface-defect-detection-model-service/app/routers/predict.py

```
@@ -0,0 +1,198 @@
+    confidence_threshold: Optional[float] = Field(0.5, ge=0.0, le=1.0, description="신뢰도 임계값 (0.0-1.0)")
+    
+    @validator('image_base64')
+    def validate_base64(cls, v):
+        """Base64 데이터 유효성 검사"""
+        if not v or len(v) < 100:
+            raise ValueError("Base64 이미지 데이터가 유효하지 않습니다")
+        try:
+            import base64
+            base64.b64decode(v)
+        except Exception:
+            raise ValueError("유효하지 않은 Base64 인코딩입니다")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)